src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -145,13 +167,22 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
             return;
         }
 
+        let formContent;
+        if (contingencyListType === ContingencyListType.FORM) {
+            formContent = currentCriteriaBasedFilter.current;
+        } else if (contingencyListType === ContingencyListType.SCRIPT) {
+            formContent = currentScript;
```

Comment:
maybe :
formContent = {script: currentScript} ?
So you don't need body.script = formContent in rest-api.js

Reply:
Yes